packaging/build_wheel.sh

```diff
@@ -43,10 +43,13 @@ setup_macos() {
 # Outputs:
 #   PATH modified to put correct Python version in PATH
 setup_wheel_python() {
-  eval "$(conda shell.bash hook)"
-  conda env remove -n "env$PYTHON_VERSION" || true
-  conda create -yn "env$PYTHON_VERSION" python="$PYTHON_VERSION"
-  conda activate "env$PYTHON_VERSION"
+  if [[ -n "$PYTHON_VERSION" ]]; then
+    eval "$(conda shell.bash hook)"
+    conda env remove -n "env$PYTHON_VERSION" || true
+    conda create -yn "env$PYTHON_VERSION" python="$PYTHON_VERSION"
+    conda activate "env$PYTHON_VERSION"
+ fi
```

Comment:
Noob question: Do we need to install `conda` before this step?

Reply:
nit: the indentation of this line look incorrect

Reply:
@ejguan If you want to use Conda to set up Python environment, then yes. So that's why I say "if PYTHON_VERSION is unset, don't use conda.."